src/elf/sym.rs

```diff
@@ -468,6 +483,7 @@ if_alloc! {
         type Item = <SymIterator<'a> as Iterator>::Item;
         type IntoIter = SymIterator<'a>;
 
+        #[inline]
```

Comment:
This is the only one that I don't see any benefit for, since it won't be called often and there's little room for optimization.